Platforms/QemuQ35Pkg/.azurepipelines/Windows-VS.yml

```diff
@@ -48,6 +49,22 @@ jobs:
             Run.Flags: $(run_flags)
             Run: $(should_run)
 
+          QemuQ35_DEBUG_NO_SMM:
+            Build.File: "Platforms/$(package)/PlatformBuild.py"
+            Build.Arch: $(arch_list)
+            Build.Flags: "BLD_*_SMM_ENABLED=FALSE"
+            Build.Target: "DEBUG"
+            Run.Flags: "$(run_flags) BLD_*_SMM_ENABLED=FALSE"
+            Run: $(should_run)
+
+          QemuQ35_RELEASE_NO_SMM:
+            Build.File: "Platforms/$(package)/PlatformBuild.py"
+            Build.Arch: $(arch_list)
+            Build.Flags: "BLD_*_SMM_ENABLED=TRUE"

```

Comment:
Was `Build.Flags` including `SMM_ENABLED` set to `TRUE` a mistake here (config is `QemuQ35_RELEASE_NO_SMM`)?